internal/xcuitest/config_test.go

```diff
@@ -15,6 +15,35 @@ import (
 	"gotest.tools/v3/fs"
 )
 
+func TestToMap(t *testing.T) {
```

Comment:
It'd be better to organize this test like other unit tests, e.g. Define test table and test data structure.

Reply:
There's only one test scenario though. Seemed a bit overkill to have a test table with just one test case.